### PR TITLE
PR template update: please add to changelog

### PR DESCRIPTION
**Pull request by metachris**
Added a line to the PR template about adding something to CHANGELOG

### CODE REVIEW

**Comment by metachris** [comment]
Good point, updated the PR.

**Comment by localhuman** [comment]
Would it make sense to add a checklist to the template instead of a series of Y/N questions?

**Comment by metachris** [comment]
I think it would. Updated the PR

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
-**Did you run `make lint` and `make test`?**
-
-**Are you making a PR to a feature branch or development rather than master?**
+**Please check the following, if applicable:**
 
-**Did you add an entry to `CHANGELOG.rst`? (if not, please do)**
+- [ ] Did you add any tests?
+- [ ] Did you run `make lint`?
+- [ ] Did you run `make test`?
+- [ ] Are you making a PR to a feature branch or development rather than master?
```
Comment (ixje) [inline .github/PULL_REQUEST_TEMPLATE.md]: this question cannot be handled by a check box

Comment (metachris) [reply .github/PULL_REQUEST_TEMPLATE.md]: yes, it can, it still is a yes/no question.

Comment (ixje) [reply .github/PULL_REQUEST_TEMPLATE.md]: After reading it again I guess we do not differentiate between Dev vs feature?

Comment (ixje) [reply .github/PULL_REQUEST_TEMPLATE.md]: ok nm

**Pull request merged**